src/web/src/components/SearchInput.tsx

```diff
@@ -49,7 +49,7 @@ export const SearchInput: React.FC<InputProps> = ({
       <div className="join">
         <input
           type="search"
-          className={`input join-item input-sm w-full border-0 focus:outline-0 ${heightOverride}`}
+          className={`input join-item input-xs !h-[38px] w-full border-0 !pl-4 placeholder-[#858585] focus:outline-0 ${heightOverride}`}
```

Comment:
please note that this control can be used from multiple places such as admin organization/opportunity search etc

hardcoded colors may not fit in with the current theme (bg-theme) i.e
admins - blue
org admin - green

it will be better to use the color from the theme (bg-theme) together with brightness-x to augment the colour.

Reply:
Yes meneer, I am aware. Only adjusting the placeholder color here to match the rest of the inputs